code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -303,54 +348,170 @@
 		qdel(cavity_item)
 	return ..()
 
+/obj/item/bodypart/chest/drop_organs(mob/user)
+	if(cavity_item)
+		cavity_item.forceMove(user.loc)
+		cavity_item = null
+	..()
```

Comment:
In the unlikely scenario that someone makes a chest bodypart actually drop outside a mob, then opening it with sharp tools drop its cavity item.